src/manifest.ts

```diff
@@ -48,8 +48,67 @@ export class SlackManifest {
           messages_tab_read_only_enabled: true,
         },
       },
-      outgoing_domains: def.outgoingDomains || [],
-    };
+      settings: { "function_runtime": this.getFunctionRuntime() },
+    } as ManifestSchema;
+
+    if (def.slackHosted === false && def.slackHosted !== undefined) {
+      //Settings
+      if (def.settings === undefined) {
+        manifest.settings = {};
+      }
+      if (def.settings !== undefined) {
+        manifest.settings = def.settings;
+      }
+      manifest.settings.function_runtime = this.getFunctionRuntime();
+      if (def.eventSubscriptions !== undefined) {
+        manifest.settings.event_subscriptions = def.eventSubscriptions;
+      }
+      if (def.socketModeEnabled !== undefined) {
+        manifest.settings.socket_mode_enabled = def.socketModeEnabled;
+      }
+      if (def.tokenRotationEnabled !== undefined) {
+        manifest.settings.token_rotation_enabled = def.tokenRotationEnabled;
+      }
+
+      //AppDirectory
+      if (def.appDirectory !== undefined) {
+        manifest.app_directory = def.appDirectory;
+      }
+
+      //OauthConfig
+      if (def.userScopes !== undefined) {
+        manifest.oauth_config.scopes.user = def.userScopes;
+      }
+      if (def.redirectUrls !== undefined) {
+        manifest.oauth_config.redirect_urls = def.redirectUrls;
+      }
+      if (def.tokenManagementEnabled !== undefined) {
+        manifest.oauth_config.token_management_enabled =
+          def.tokenManagementEnabled;
+      }
+      //Features
+      if (def.features?.appHome !== undefined) {
+        manifest.features.app_home = def.features?.appHome;
+      }
```

Comment:
It created conflicts since the `ManifestFeaturesSchema` in ManifestSchema was still including the old app home schema. Updated it with `ManifestAppHomeSchema`, so the logic added on the last release to AppHome became common in both slack-hosted and remote types. Also, existing logic in [this file ](https://github.com/slackapi/deno-slack-sdk/blob/expand-manifest-types-fekiz/src/manifest.ts#L153) updated with `home-tab_enabled` addition to override any existing `app_home` defaults if supplied in the manifest. And, failing test cases due to naming convention inconsistencies such as using camelCase with defining lower-level properties fixed. Camel case will only be in use at the upper level `ISlackManifestRemote` & `ISlackManifestHosted` types. Other Schema types as well as ManifestSchema are named using snake case.